2024-07-12 13:29:17.908 [info] (Client) Pylance async client (2024.7.1) started with python extension (2024.10.0)
2024-07-12 13:29:18.661 [info] [Info  - 1:29:18 PM] (899) Pylance language server 2024.7.1 (pyright version 1.1.369, commit 1168be71) starting
2024-07-12 13:29:18.661 [info] [Info  - 1:29:18 PM] (899) Server root directory: file:///home/ndao/.vscode-server/extensions/ms-python.vscode-pylance-2024.7.1/dist
2024-07-12 13:29:18.675 [info] [Info  - 1:29:18 PM] (899) Starting service instance "ndao"
2024-07-12 13:29:18.760 [info] [Info  - 1:29:18 PM] (899) Setting pythonPath for service "ndao": "/bin/python3"
2024-07-12 13:29:18.760 [info] [Info  - 1:29:18 PM] (899) Setting environmentName for service "ndao": "3.8.10 (global)"
2024-07-12 13:29:18.761 [info] [Info  - 1:29:18 PM] (899) No include entries specified; assuming /home/ndao
2024-07-12 13:29:18.762 [info] [Info  - 1:29:18 PM] (899) Auto-excluding **/node_modules
2024-07-12 13:29:18.763 [info] [Info  - 1:29:18 PM] (899) Auto-excluding **/__pycache__
2024-07-12 13:29:18.764 [info] [Info  - 1:29:18 PM] (899) Auto-excluding **/.*
2024-07-12 13:29:18.826 [info] [Info  - 1:29:18 PM] (899) Assuming Python version 3.8.10.final.0
2024-07-12 13:29:18.989 [info] [Info  - 1:29:18 PM] (899) No source files found.
2024-07-12 13:29:19.740 [info] [Info  - 1:29:19 PM] (899) Background analysis(1) root directory: file:///home/ndao/.vscode-server/extensions/ms-python.vscode-pylance-2024.7.1/dist
2024-07-12 13:29:19.741 [info] [Info  - 1:29:19 PM] (899) Background analysis(1) started
